### PR TITLE
Removed memcpy before call av1_inv_transform_recon() in inv_transform_recon_copy()

### PR DESCRIPTION
**Pull request by spawlows**

### CODE REVIEW

File: Source/Lib/Common/Codec/EbFullLoop.c [hunk 1]
```diff
@@ -2434,7 +2432,7 @@ void product_full_loop_tx_search(
 
             if (context_ptr->spatial_sse_full_loop) {
                 if (yCountNonZeroCoeffsTemp)
-                    inv_transform_recon_copy(
+                    inv_transform_recon_wraper(
```
Comment (joelsole) [inline Source/Lib/Common/Codec/EbFullLoop.c]: wrapper?

Comment (spawlows) [reply Source/Lib/Common/Codec/EbFullLoop.c]: Yes, thanks, corrected.

**Comment by spawlows** [comment]
Average speed gain on Windows is ~0.9%

**Comment by joelsole** [comment]
Could you rebase to master?

**Pull request merged**